saleor/core/tasks.py

```
@@ -11,7 +11,8 @@
 
 @app.task
 def delete_from_storage_task(path):
-    default_storage.delete(path)
+    if default_storage.exists(path):
+        default_storage.delete(path)
```

Django is already silently skipping the file if it doesn't exist upon deletion failure

---

I drop removing file URL when the attribute value is deleted.

---

The `exists()` check can be removed as Django is already skipping inexistent files, it will save a 3rd party call